Tools/autotest/autotest.py

```diff
@@ -330,7 +330,10 @@ def binary_path(step, debug=False):
             config_name = __bin_names[vehicle].split(".")[0]
             binary_name = __bin_names[vehicle].split(".")[1]
         else:
-            config_name = 'sitl'
+            if opts.sitl_32bit:
+                config_name = 'sitl_32bit'
```

Comment:
i'd strongly prefer this not be a separate build directory

Reply:
Done.